bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -114,7 +114,7 @@ PreProcessor::PreProcessor(shared_ptr<MsgsCommunicator> &msgsCommunicator,
                            metricsComponent_.RegisterCounter("preProcessRequestTimedout"),
                            metricsComponent_.RegisterCounter("preProcReqSentForFurtherProcessing"),
                            metricsComponent_.RegisterCounter("preProcPossiblePrimaryFaultDetected"),
-                           metricsComponent_.RegisterGauge("currentPreProcReqNum", 0)},
+                           metricsComponent_.RegisterGauge("PreProcCurrentReqNum", 0)},
```

Comment:
preProcCurrentReqNum sounds like a "sequence number of current request", which is confusing.

PreProcCurrentReqNum -> preProcInFlyRequestsNum

Reply:
fixed